code/modules/paperwork/photocopier.dm

```diff
@@ -120,7 +120,7 @@
 		else if(doccopy)
 			for(var/i = 0, i < copies, i++)
 				if(toner > 5 && !busy && doccopy)
-					new /obj/item/documents/photocopy(src, doccopy)
+					new /obj/item/documents/photocopy(loc)

```

Comment:
protip: maybe comment with "the cargo export code already checks for multiple copies" instead of being a cunt about fairly new, sparsely promoted code